sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -74,7 +76,6 @@ static az_mqtt5_connect_data test_mqtt5_connect_data = {
 
 static az_mqtt5_options options = {
```

Comment:
Should this struct be defined within separate ifdefs for the different mqtt clients if they have different values?

Reply:
I can add that once we make use of the different parts of the structure, we currently don't use the `openssl_engine` in az_mosquitto's `az_mqtt_options` nor do we test it